### PR TITLE
Add new reason code field to the chargeback api doc

### PR DESCRIPTION
**Pull request by MollieMateus**

### CODE REVIEW

File: source/reference/v2/chargebacks-api/get-chargeback.rst [hunk 1]
```diff
@@ -119,6 +119,29 @@ Response
 
      - The date and time the chargeback was issued, in `ISO 8601 <https://en.wikipedia.org/wiki/ISO_8601>`_ format.
 
+   * - ``reason``
+
+       .. type:: object|null
+
+     - Reason for the chargeback.
+
+       This field will only be returned in the API if the original payment method supports chargeback reasons.
```
Comment (MollieRick) [inline source/reference/v2/chargebacks-api/get-chargeback.rst:128]: I would change it to;
`Reason for the chargeback as given by the bank`

with a notice that it is only available for `directdebit`.

File: source/reference/v2/chargebacks-api/get-chargeback.rst [hunk 2]
```diff
@@ -247,6 +270,10 @@ Response
            "value": "-35.07"
        },
        "createdAt": "2018-03-14T17:00:52.0Z",
+        "reason": {
+          "code": "MS03",
+          "description": "Reason not specified"
```
Comment (MollieRick) [inline source/reference/v2/chargebacks-api/get-chargeback.rst:275]: I think a better example would be nice. "Reason not specified" is no 'muh' for in the docs :D

**Review by MollieRick — CHANGES_REQUESTED** [review]

File: source/reference/v2/chargebacks-api/get-chargeback.rst (hunk 2 above)
Comment (MollieMateus) [reply source/reference/v2/chargebacks-api/get-chargeback.rst]: I agree =D

File: source/reference/v2/chargebacks-api/get-chargeback.rst (hunk 1 above)
Comment (MollieMateus) [reply source/reference/v2/chargebacks-api/get-chargeback.rst]: Good

**Comment by MollieMateus** [comment]
@MollieRick documentation updated 😄

File: source/reference/v2/chargebacks-api/get-chargeback.rst [hunk 3]
```diff
@@ -119,6 +119,29 @@ Response
 
      - The date and time the chargeback was issued, in `ISO 8601 <https://en.wikipedia.org/wiki/ISO_8601>`_ format.
 
+   * - ``reason``
+
+       .. type:: object|null
+
+     - Reason for the chargeback as given by the bank.
+
+       This field will only be returned for chargebacks where *direct debit* was used as the original payment method.
```
Comment (MollieRick) [inline source/reference/v2/chargebacks-api/get-chargeback.rst:128]: NBC: Maybe put it in a `..note::` field :)

**Review by MollieRick — APPROVED** [review]

File: source/reference/v2/chargebacks-api/get-chargeback.rst (hunk 3 above)
Comment (MollieMateus) [reply source/reference/v2/chargebacks-api/get-chargeback.rst:128]: Didn't know this thing existed hehehe. But I think it's a great idea.

**Pull request merged**